codebuild/spec/buildspec_generalbatch.yml

```diff
@@ -367,3 +367,11 @@ batch:
         compute-type: BUILD_GENERAL1_LARGE
         image: aws/codebuild/standard:7.0
         privileged-mode: true
+    - identifier: ktlsASAN
+      buildspec: codebuild/spec/buildspec_ktls.yml
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+        image: aws/codebuild/standard:7.0
+        privileged-mode: true
+        variables:
+            S2N_CMAKE_OPTIONS: "-DASAN=ON"
```

Comment:
This environment variable isn't making it into the VM, based on the Cmake banner.  While there is an option to allow env. variable forwarding with ssh, it's a bit of work... it might be faster to add an `export S2N_CMAKE_OPTIONS=${S2N_CMAKE_OPTIONS}` as part of  the nix command in the ktls buildspec at https://github.com/aws/s2n-tls/blob/731860989848edc0cde33324a3141fc7d0116ff0/codebuild/spec/buildspec_ktls.yml#L20

Reply:
Yeah I definitely didn't do sufficient testing :( I fixed this and I'll update the testing section of the description with better manual testing.